packages/migrate/src/types.ts

```diff
@@ -149,10 +152,12 @@ export namespace EngineArgs {
     compositeTypeDepth?: number
     schemas?: string[]
   }
+
   export interface IntrospectResult {
     datamodel: string
     warnings: IntrospectionWarnings[]
     version: IntrospectionSchemaVersion
+    views?: IntrospectionViewDefinition[]
```

Comment:
I think this will always be included now, as (in the words of Julius) "the protocol doesn't let us to have optional keys, the key has to be there".